### PR TITLE
fix(accessibility): improve output for long elements

### PR DESCRIPTION
**Pull request by juliemr**
Instead of just printing the first N characters of the element's template,
print the first and last N/2.

See #1854

### CODE REVIEW

File: plugins/accessibility/index.js [hunk 1]
```diff
@@ -175,6 +175,15 @@ function runChromeDevTools(config) {
   var elementPromises = [],
       elementStringLength = 200;
 
+  function trimText(text) {
+    if (text.length > elementStringLength) {
+      return text.substring(0, elementStringLength / 2) + ' ... '
```
Comment (sjelin) [inline plugins/accessibility/index.js]: This will have a total length of `elementStringLength + 5`.  Is that correct?

Comment (juliemr) [inline plugins/accessibility/index.js]: Yeah (there's also some tabs at the beginning, so it's not the whole output string length anyway...).

**Comment by sjelin** [comment]
LGTM

**Comment by marcysutton** [comment]
:+1:

**Comment by juliemr** [comment]
Thanks! Merged as fb92be6d588e7302989bd171a064739359bb3c74

**Pull request closed without merging**